app/javascript/components/track/iteration-summary/AnalysisStatusSummary.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import { RepresentationStatus, AnalysisStatus } from '../IterationSummary'
+
+function Content({
+  analysisStatus,
+  representationStatus,
+}: {
+  analysisStatus: AnalysisStatus
+  representationStatus: RepresentationStatus
+}) {
+  if (
+    analysisStatus === AnalysisStatus.QUEUED ||
+    representationStatus === RepresentationStatus.QUEUED
+  ) {
+    return (
+      <>
+        <svg role="presentation" className="icon">
+          <use xlinkHref="#loading" />
+        </svg>
+        <div className="status">Analysing</div>
+      </>
+    )
+  }
+
+  return (
+    <>
+      <div className="dot"></div>
+      <div className="status">Analysed</div>
+    </>
+  )
+}
+
+export function AnalysisStatusSummary({
+  analysisStatus,
+  representationStatus,
+}: {
+  analysisStatus: AnalysisStatus
+  representationStatus: RepresentationStatus
+}) {
+  return (
+    <div className="analyzer">
```

Comment:
```suggestion
    <div className="analyzer" role="status">
```

This will _announce_ changes to this element!  So when it's analysed, it will read that. [role="status"](https://www.w3.org/TR/wai-aria/#status). It has an implicit value of `aria-live="polite"`.

Reply:
This does mean that if you want to re-use the component, it should have an id passed in, or the `id` on the page won't be unique. Not important for now.